packages/web3-utils/src/converters.ts

```diff
@@ -100,14 +100,9 @@ export const hexToBytes = (bytes: HexString): Buffer => bytesToBuffer(bytes);
 export const hexToNumber = (value: HexString): bigint | number => {
 	validator.validate(['hex'], [value]);
```

Comment:
Should this be moved to `web3-validator`'s implementation?

Reply:
Yes we should. I will fix it.

Reply:
@spacesailor24 Just verified `hexToNumber` is already using `web3-validator` implementation.

Reply:
I was thinking of moving 
`validator.validate(['hex'], [value]);` to `web3-validator`'s implementation of `hexToNumber` so that this wrapper method is as simple as possible